### ISSUE TITLE
Desktop: System.Security.Cryptography.Encryption.Aes.Tests.DecryptorReusabilty.TestDecryptorReusability failed with "Xunit.Sdk.EqualException"

### ISSUE THREAD

**Issue by Jiayili1** [dotnet/corefx#18903]
Failed test: System.Security.Cryptography.Encryption.Aes.Tests.DecryptorReusabilty.TestDecryptorReusability

Detail: https://ci.dot.net/job/dotnet_corefx/job/master/job/outerloop_netfx_windows_nt_debug/30/testReport/System.Security.Cryptography.Encryption.Aes.Tests/DecryptorReusabilty/TestDecryptorReusability/

MESSAGE:
~~~
Assert.Equal() Failure
    Expected: Byte[] [20, 48, 113, 173, 237, ...]
    Actual: Byte[] [20, 15, 15, 16, 17, ...]
~~~
STACK TRACE:
~~~
at System.Security.Cryptography.Encryption.Aes.Tests.DecryptorReusabilty.TestDecryptorReusability() 
in D:\j\workspace\outerloop_net---903ddde6\src\Common\tests\System\Security\Cryptography\AlgorithmImplementations\AES\DecryptorReusability.cs:line 65
~~~

**Comment by danmosemsft** [comment]
@bartonjs @krwq

**Comment by bartonjs** [comment]
If this is AesCryptoServiceProvider that's a bug that was fixed as a retargeting change in 4.6.2.  So maybe for netfx we'd want to measure the result based off of the `Switch.System.Security.Cryptography.AesCryptoServiceProvider.DontCorrectlyResetDecryptor` switch.
